external-crates/move/crates/move-analyzer/trace-debug/README.md

```diff
@@ -1,20 +1,42 @@
-```bash
-npm install --prefix ../move-trace-adapter
-```
-3. Package the extension (it will pull relevant files from the DAP implementation)
-```
-vsce package -o move-trace-debug.vsix
+1. Open a new window in any Visual Studio Code application version 1.61.0 or greater.
+2. Open the command palette (`⇧` + `⌘` + `P` on macOS, or use the menu item *View > Command Palette...*) and
+   type **Extensions: Install Extensions**. This will open a panel named *Extensions* in the
+   sidebar of your Visual Studio Code window.
+3. In the search bar labeled *Search Extensions in Marketplace*, type **Move Trace**. The Move Trace debugger extension
+   should appear as one of the option in the list below the search bar. Click **Install**.
+
+# How to trace-debug a Move unit test
+
+Debugging a Move unit tests consists of two steps: generating a Move trace and actually trace-debugging it
+
+## Generating a Move trace
+
+If you have [Mysten's Move extension](https://marketplace.visualstudio.com/items?itemName=mysten.move) installed you can generate a Move trace for tests defined in a given file by navigating to this file in VSCode and running `Move: Trace Move test execution` from the command palette. See the description of [Mysten's Move extension](https://marketplace.visualstudio.com/items?itemName=mysten.move) for pre-requisites needed to run this command.
+
+If you plan to use the the Trace Debugging Extension by itself, you need to generate the traces using command-line interface of `sui` binary. See [here](https://docs.sui.io/guides/developer/getting-started/sui-install) for instructions on how to install `sui` binary. Note that the `sui` binary must be built with the `tracing` feature flag. If your version of the `sui` binary was not built with this feature flag, an attempt to trace test execution will fail. In this case you may have to build the `sui` binary from source following these [instructions](https://docs.sui.io/guides/developer/getting-started/sui-install#install-sui-binaries-from-source).
+
+Once the `sui` binary is installed, you generate traces for all test files in a given package by running the following command in the package's root directory:
+```shell
+sui move test OPTIONAL_FILTER --trace-execution
 ```
-4. Install move-trace-debug.vsix in VSCode using `Install from VSIX...` option
+
+The `OPTIONAL_FILTER` of the command provides a way to limit trace generation to the tests whose name contains the `OPTIONAL_FILTER` string.
+
+## Trace-debugging a test
+
+Once traces are generated, open a file containing the test you want to trace-debug and execute `Run->Start Debug` command. The first time you execute this command, you will have to choose the debugging configuration for Move files, of which there should be only one available. Then you will have to choose a test to trace-debug if there is more than one test in a file (otherwise a trace-debugging sessio for a single test will start automatically).
```

Comment:
This is the Move file, just to be clear?

Reply:
Yes. Added clarification to the text